test/Concurrency/nonisolated_access.swift

```diff
@@ -39,6 +41,8 @@ actor A {
   func test() {
     var s = S()
     s.x += 0 // okay
+    // expected-error@+1 {{main actor-isolated property 'y' can not be mutated on a nonisolated actor instance}}
```

Comment:
Just bringing to notice that this error is confusing — the instance is a struct, not an actor.

Reply:
Yea, good point. I read this as actor instance is referring to actor A, but I can see your point upon a closer look

Reply:
#76909.